src/app/components/translation/translation-version-badge/translation-version-badge.component.html

```diff
@@ -1,9 +1,12 @@
 <span
   class="badge badge-secondary"
-  *ngIf="!translation.is_published && !translation.none"
+  *ngIf="!translation.is_published && !translation.none && !translation['publishing-errors']"
   >{{ translation.version }} | Draft</span
 >
-<span class="badge badge-success" *ngIf="translation.is_published"
+<span class="badge badge-success" *ngIf="translation.is_published && !translation['publishing-errors']"
```

Comment:
in theory if it's published it shouldn't actually have a publishing-error

Reply:
I pushed this to the wrong PR, so I'm going to remove it and add it to the new publishing PR